plotly/tools.py

```diff
@@ -1443,6 +1443,376 @@ class FigureFactory(object):
     """
 
     @staticmethod
+    def create_scatterplotmatrix(df, useindex = False, index = " ",
```

Comment:
Can you use `None`, not `" "` as default values for these things? `" "` is a truthy value, which is confusing, IMO.